File: clients/csharp/examples/tables-drop-type.cs
using DataStax.AstraDB.DataApi;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    // Get an existing table
    var client = new DataAPIClient();
    var database = client.GetDatabase(
      "**API_ENDPOINT**",
      "**APPLICATION_TOKEN**"
    );

    // Drop a user-defined type
    await database.DropTypeAsync("**UDT_NAME**");
  }
}

File: clients/java/examples/tables-drop-type.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient("**APPLICATION_TOKEN**").getDatabase("**API_ENDPOINT**");

    // Drop a user-defined type
    database.dropType("**UDT_NAME**");
  }
}
